sunpy/map/sources/soho.py

```diff
@@ -74,8 +74,11 @@ def __init__(self, data, header, **kwargs):
     @property
     def rsun_obs(self):
         """
-        Meta keyword : ``solar_r``, ``cdelt1``.
         Returns the solar radius as measured by EIT in arcseconds.
+        .. rubric:: Metadata keyword:
+
+        * ``solar_r``
```

Comment:
This is an actual operation on two keywords so we should probably write it as ``solar_r * cdelt1 `` to make it clear that's what it is.